test/test_unary_ufuncs.py

```diff
@@ -494,6 +511,18 @@ def test_sqrt_complex_edge_values(self, device, dtype):
         x = torch.tensor(-1.0000e+20 - 4988429.2000j, dtype=dtype, device=device)
         self.compare_with_numpy(torch.sqrt, np.sqrt, x)
 
+    @dtypes(torch.cfloat, torch.cdouble)
+    def test_abs_complex_edge_values(self, device, dtype):
+        values = get_hard_complex_values(dtype)
+
+        for value in values:
+            t = torch.tensor(value, dtype=dtype, device=device)
+            self.compare_with_numpy(torch.abs, np.abs, t)
+
+            # for vectorized version.
+            t = torch.tensor([value] * 100, dtype=dtype, device=device)
```

Comment:
*8 should do it. 

Another way to test this would be to acquire the values and construct a tensor from them. Then the values of that tensor can be viewed individually to be given as input to the non-vectorized test, then that tensor can be catted with itself and passed to test the vectorized version.

This approach would greatly reduce the number of tensor creations and cuda kernel invocations in this test. What would you think about trying it?